cosmos/testing/integration/fixture.go

```diff
@@ -132,6 +140,38 @@ func (tf *TestFixture) CreateKeyWithName(name string) {
 	tf.keysMap[name] = newKey
 }
 
+func (tf *TestFixture) SendGraphQLRequest(query string) (string, int, error) {
+	url := tf.HTTPAddr + "/graphql"
+	requestBody, err := json.Marshal(map[string]string{
+		"query": query,
+	})
+	if err != nil {
+		return "", 500, err
+	}
+
+	client := &http.Client{}
+
+	req, err := http.NewRequest(http.MethodPost, url, bytes.NewBuffer(requestBody))
+	if err != nil {
+		return "", 500, err
+	}
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := client.Do(req)
```

Comment:
rather than rewriting http connection logic, try using the Geth's built-in rpc client 
https://github.com/berachain/polaris-geth/blob/stateful-v1.11.4/rpc/client.go#L166

Reply:
GraphQL is an alternative to RPC, won't work i believe

https://eips.ethereum.org/EIPS/eip-1767